ajax/libs/multi.js/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "multi.js",
+  "filename": "multi.min.js",
+  "version": "0.2.3",
+  "description": "A user-friendly replacement for select boxes with multiple attribute enabled",
+  "keywords": [
+    "user-friendly",
+    "select",
+    "multiple",
+    "simple"
+  ],
+  "author": {
+    "name": "Fabian Lindfors",
+    "url": "https://fabianlindfors.se"
```

Comment:
Let's use `https` because this is more safety for browsing.
https://fabianlindfors.se/
Thanks.

Reply:
It is https

Reply:
OK